sycl/include/CL/__spirv/spirv_ops.hpp

```diff
@@ -928,6 +929,14 @@ __CLC_BF16_SCAL_VEC(uint32_t)
 #undef __CLC_BF16_SCAL_VEC
 #undef __CLC_BF16
 
+__SYCL_CONVERGENT__ extern SYCL_EXTERNAL __SYCL_EXPORT __spv::complex_half
+__spirv_GroupCMulKHR(unsigned int, unsigned int, __spv::complex_half) noexcept;
+__SYCL_CONVERGENT__ extern SYCL_EXTERNAL __SYCL_EXPORT __spv::complex_float
+__spirv_GroupCMulKHR(unsigned int, unsigned int, __spv::complex_float) noexcept;
+__SYCL_CONVERGENT__ extern SYCL_EXTERNAL __SYCL_EXPORT __spv::complex_double
+__spirv_GroupCMulKHR(unsigned int, unsigned int,
```

Comment:
AFAIK there no KHR extension for these, you should change for `GroupCMulINTEL`

Reply:
Good catch. Fixed